tests/unittests/test_validators.py

```diff
@@ -86,3 +88,17 @@ def test_validate_area(feature_germany_heidelberg):
 def test_validate_area_exception(feature_germany_heidelberg):
     with pytest.raises(SizeRestrictionError):
         validate_area(feature_germany_heidelberg)
+
+
+def test_wrong_crs():
+    feature = get_geojson_fixture("heidelberg-altstadt-epsg32632.geojson")
+    try:
+        validate_geojson(feature)
+    except InvalidCRSError as exception:
+        print("test")
+        assert (
+            str(exception.message)
+            == "Invalid CRS. The FeatureCollection must have the EPSG:4326 CRS or none."
+        )
+    else:
+        raise AssertionError("Expected InvalidCRSError but no exception was raised")
```

Comment:
I suggest using pytest functionality to test if a assertion gets raise: https://docs.pytest.org/en/7.1.x/getting-started.html#assert-that-a-certain-exception-is-raised